test/storage/bucket.js

```diff
@@ -623,6 +727,236 @@ describe('Bucket', function() {
+      var fakeNextQuery = { a: 'b', c: 'd' };
+
+      bucket.getFiles = function(query, callback) {
+        if (Object.keys(query).length === 0) {
+          // First time through, return a `nextQuery` value.
+          callback(null, [], fakeNextQuery);
+        } else {
+          // Second time through.
+          assert.deepEqual(query, fakeNextQuery);
+          done();
+        }
+      };
+
+      bucket.makeAllFilesPublicPrivate_({}, assert.ifError);
+    });
+
+    it('should execute callback with error from getting files', function(done) {
+      var error = new Error('Error.');
+
+      bucket.getFiles = function(query, callback) {
+        callback(error);
+      };
+
+      bucket.makeAllFilesPublicPrivate_({}, function(err) {
+        assert.equal(err, error);
+        done();
+      });
+    });
+
+    it('should execute callback with error from changing file', function(done) {
+      var error = new Error('Error.');
+
+      var files = [
+        bucket.file('1'),
+        bucket.file('2')
+      ].map(util.propAssign('makePublic', function(callback) {
+        callback(error);
+      }));
+
+      bucket.getFiles = function(query, callback) {
+        callback(null, files);
+      };
+
+      bucket.makeAllFilesPublicPrivate_({ public: true }, function(err) {
+        assert.equal(err, error);
+        done();
+      });
+    });
+
+    it('should execute callback with queued errors', function(done) {
+      var error = new Error('Error.');
+
+      var files = [
+        bucket.file('1'),
+        bucket.file('2')
+      ].map(util.propAssign('makePublic', function(callback) {
+        callback(error);
+      }));
+
+      bucket.getFiles = function(query, callback) {
+        callback(null, files);
+      };
+
+      bucket.makeAllFilesPublicPrivate_({
+        public: true,
+        force: true
+      }, function(errs) {
+        assert.deepEqual(errs, [error, error]);
+        done();
+      });
+    });
+
+    it('should execute callback with files changed', function(done) {
+      var error = new Error('Error.');
+
+      var successFiles = [
+        bucket.file('1'),
+        bucket.file('2')
+      ].map(util.propAssign('makePublic', function(callback) {
+        callback();
+      }));
+
+      var errorFiles = [
+        bucket.file('3'),
+        bucket.file('4')
+      ].map(util.propAssign('makePublic', function(callback) {
+        callback(error);
+      }));
+
+      bucket.getFiles = function(query, callback) {
+        callback(null, successFiles.concat(errorFiles));
+      };
+
+      bucket.makeAllFilesPublicPrivate_({
+        public: true,
+        force: true
+      }, function(errs, files) {
+        assert.deepEqual(errs, [error, error]);
+        assert.deepEqual(files, successFiles);
+        done();
+      });
+    });
+  });
+
+  describe('setPredefinedAcl_', function() {
+    it('should make correct public PATCH request', function(done) {
```

Comment:
I can remove the `setPredefinedAcl_` function altogether since we're just using it for `private` now.